lib/components/button.vue

```diff
@@ -1,87 +1,96 @@
 <template>
-    <button :class="classList"
+    <button v-bind="conditionalLinkProps"
```

Comment:
This is a neat trick.  I never realized one could bind multiple props/values in a single v-bind (until I just read the docs :wink: )